http-server/src/main/java/org/triplea/server/lobby/game/listing/GameListingEventQueue.java

```diff
@@ -25,36 +21,19 @@
   private final BiConsumer<Collection<Session>, ServerMessageEnvelope> broadcaster;
 
   @Getter(value = AccessLevel.PACKAGE, onMethod_ = @VisibleForTesting)
-  private final Map<String, Session> sessions = new HashMap<>();
+  private final SessionSet sessionSet;
 
   void addListener(final Session session) {
     Preconditions.checkNotNull(session);
     Preconditions.checkNotNull(session.getId());
```

Comment:
Now that the ID is no longer required, we don't need it to be non-null

Reply:
Removed: 52b859842